app/services/streaming_service.py

```
@@ -0,0 +1,707 @@
+        # Send final chunk only if not already sent
+        if buffer and not buffer.is_closed() and not stream.final_sent:
+            stream.final_sent = True
+            final_chunk = StreamChunk(
+                data={"status": "complete", "error": error},
+                sequence=stream.chunks_sent,
+                is_final=True
+            )
+            try:
+                await buffer.put(final_chunk)
+            except Exception:
+                pass
```

Ruff (0.14.14)

`try`-`except`-`pass` detected, consider logging the exception

(S110)